packages/flex-plugins-api-toolkit/src/scripts/describePlugin.ts

```diff
@@ -61,23 +66,24 @@ export default function describePlugin(
     ]);
 
     let isPluginActive = false;
-    const formattedVersions: PluginVersion[] = versions.plugin_versions.map((version) => ({
+    const formattedVersions: PluginVersion[] = versions.plugin_versions.map((version: PluginVersionResource) => ({
       sid: version.sid,
       version: version.version,
       url: version.plugin_url,
       changelog: version.changelog,
       isPrivate: version.private,
       isActive: false,
+      isArchived: version.archived,
```

Comment:
should this be `version.archived || plugin.archived` since if a plugin is archived then all of its versions are not usable?

Reply:
That would automatically be done in the API layer. So `version.archived` would be true from the API response if the plugin is archived.

Reply:
ahh okay nice